repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -526,6 +526,13 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
                 request.addHeader(OpenStackServiceHeader.EXTENDED_AUTHORIZATION.toString, X_AUTH_PROXY)
             }
         }
+
+        // Construct and add authenticatedBy, if available
+        token.authenticatedBy foreach {
+          _ foreach {
+            request.addHeader(OpenStackServiceHeader.AUTHENTICATED_BY.toString, _)
```

Comment:
I wanted to point this out. It's not consistent with the rest of this filter which generally adds multiple headers on the same line. Perhaps, however, it is the way we should be doing it... since Repose splits headers coming in, and the merge-header filter exists if users want merged headers.

That is, this is inconsistent with the filter, but more consistent with Repose as a system, in my opinion.

Let me know what you want to do.